src/mscorlib/src/System/String.cs

```diff
@@ -411,7 +415,61 @@ private unsafe static bool EqualsHelper(String strA, String strB)
                 return (length <= 0);
             }
         }
-        
+
+        [System.Security.SecuritySafeCritical]  // auto-generated
+        [ReliabilityContract(Consistency.WillNotCorruptState, Cer.MayFail)]
+        private unsafe static bool OrdinalCompareSubstring(String strA, String strB, int length)
+        {
+            Contract.Requires(strA != null);
```

Comment:
@jkotas No it doesn't. I was thinking ahead to maybe using this with EndsWith which would be two char pointers and a length. I'll change it to the two strings until that ever happens.
@benaadams I might leave those as is for the moment just because of what I mentioned above.